Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
```

Comment:
It's necessary sometimes as it doesn't affect the output whether it's been poured from a bottle or not before (re)bottling  but it does affect the checksum.